translations/en/sources/developer.md

```diff
@@ -5,59 +5,59 @@
 
 You have the right to make mistakes and the duty to stay thirsty for knowledge!
 
-In that level you don't master programming languages, tools, development enviroment, perhaps neither the operating system that you will work. Easy... Focus on learn, take the advices of those who are longer on the area and look for absorb this knowledge.
+In this level you don't master any programming languages, tools, development envirnoments, perhaps neither the operating system that you are working with. Easy... Focus on learning, take advices from those who have been longer on the area and try to absorb this knowledge.
```

Comment:
Great @rsalgado!

Thanks a lot.

This line is `environments`. You can resolve, please?